cse110-sp24-group14/cse110-sp24-group14
# Advaith/welcome-page-skele

## Tracking Info

Resolves #18 



## Changes



**source/welcomePage/welcome.html:**

- created base page displaying statistics such as: active streaks, tasks added + completed, code snippets added

**source/welcomePage/welcome.css:**

- divided statistics into separate sections, each inside its own box

**source/welcomePage/welcome.js:**

- created a button that takes user to main page (source/index.html) when clicked on

## Testing



- Utilized live server feature on VSCode in order to:
       - make sure stats are displayed on welcome page
       - make sure button redirects user to the main page of the application

## Confirmation of Change



Base HTML page with CSS design features
<img width="1640" alt="welcome-page" src="https://github.com/cse110-sp24-group14/cse110-sp24-group14/assets/80607809/72c543db-9774-42e6-a6fc-df7a0a0d5969">

Button feature

https://github.com/cse110-sp24-group14/cse110-sp24-group14/assets/80607809/048b4e2f-0fd9-462c-aa04-69ff0c11f0ab
